localstack/services/kms/provider.py

```diff
@@ -776,7 +778,10 @@ def decrypt(
         self._validate_key_for_encryption_decryption(context, key)
         self._validate_key_state_not_pending_import(key)
 
-        plaintext = key.decrypt(ciphertext)
+        try:
+            plaintext = key.decrypt(ciphertext, encryption_context)
+        except Exception:
```

Comment:
This is a very broad exception block and not recommended. Can you be more specific with the exeception here?